templates/macros/filters/date.html

```diff
@@ -0,0 +1,20 @@
+{% macro field(name, label, dates) %}
+
+<div class="field" id="{{ name }}">
+  <label for="{{ name }}" class="label">{{ label }}</label>
+  <ul>
+    <li><input name="{{ name }}" type="radio" data-start-date={{ dates['today'][0] | date }} data-end-date={{ dates['today'][1] | date }}>Today</li>
+    <li><input name="{{ name }}" type="radio" data-start-date={{ dates['month'][0] | date }} data-end-date={{ dates['month'][1] | date }}>This month</li>
+    <li><input name="{{ name }}" type="radio" data-start-date={{ dates['quarter'][0] | date }} data-end-date={{ dates['quarter'][1] | date }}>This quarter</li>
+    <li><input name="{{ name }}" type="radio" data-start-date={{ dates['year'][0] | date }} data-end-date={{ dates['year'][1] | date }}>This year</li>
+    <li><input name="{{ name }}" type="radio">Custom</li>
+  </ul>
+  <div class="date-range-input">
+    <label for="min_date">From</label>
+    <input type="text" id="min_date" name="min_date" data-inputmask="'alias': 'mm/dd/yyyy'">
```

Comment:
Is there code that uses this inputmask?

When doing research for the pattern library, we found traditional input mask code can be problematic for screen readers: https://www.filamentgroup.com/lab/politespace.html

If we're using a traditional input mask, I suggest a discussion with Bristow to check, I would be fine merging this before that happens though.